### PR TITLE
Clarify node support + request interop

### PR DESCRIPTION
**Pull request by TehShrike**
This potential confusion came to light in #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/), [webpack](https://webpack.github.io/) etc.
 
-API is a subset of [request](https://github.com/request/request) so that the same code can be used in the browser and Node.js.
+API is a subset of [request](https://github.com/request/request) so you can write universal JS modules by using `require('request')` in your code and adding a [browser](https://github.com/substack/node-browserify#browser-field) field to your `package.json` to allow your code to work in the browser as well as Node.js:
+
+```
+"browser": {
```
Comment (naugtur) [inline README.md]: Correct me if I'm wrong, but I thought webpack is not supporting that field and you need to do it differently.

**Comment by naugtur** [comment]
Sorry I lagged with looking at it, I traveled from Europe to California and got disorganized for a couple of days.

I'm trying to get you commit access to the repo, but I'm having a hard time getting in touch with @Raynos

**Comment by Raynos** [comment]
Oh hi.

**Comment by Raynos** [comment]
I added @TehShrike as a collaborator.

File: README.md (hunk 1 above)
Comment (TehShrike) [inline README.md]: You're right,  I'll add a Webpack section too.

**Comment by TehShrike** [comment]
I added the webpack configuration option.

**Comment by TehShrike** [comment]
I'll merge now to get it into the readme, but if anyone notices anything else I'll open a new pull request.

**Comment by TehShrike** [comment]
Oh wait, somehow my collaborator-ness got lost - I think maybe all open invitations got thrown out when the repo was transferred :-x

**Comment by naugtur** [comment]
Oops. I'll fix it this evening. Sorry!

Zb

**Pull request merged**